External/YAeHMOP/CMakeLists.txt

```
@@ -2,12 +2,29 @@ if(NOT RDK_BUILD_YAEHMOP_SUPPORT)
   return()
 endif(NOT RDK_BUILD_YAEHMOP_SUPPORT)
 
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 add_definitions(-DRDK_BUILD_YAEHMOP_SUPPORT)
 
-include(ExternalProject)
+if(NOT DEFINED YAEHMOP_DIR)
+  set(YAEHMOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/yaehmop")
+endif()
+
+if(NOT EXISTS "${YAEHMOP_DIR}/tightbind/bind.h")
+    set(RELEASE_NO "2022.09.1")
+    set(MD5 "2d5ee5b6b130529544565240597c602a")
+    downloadAndCheckMD5("https://github.com/greglandrum/yaehmop/archive/refs/tags/v2022.09.1.tar.gz"
```

Does it work if we use:
```suggestion
    downloadAndCheckMD5("https://github.com/greglandrum/yaehmop/archive/refs/tags/v${RELEASE_NO}.tar.gz"
```
?

---

Yep, I actually thought I had done that. Thanks for catching it